bool has_puck
geometry_msgs/Point center
int64 color